src/scss/abstracts/_mixins.scss

```diff
@@ -93,6 +93,10 @@
 		}
 	}
 
+	// Focus/hover styles for buttons
+	.wp-block-file__button {
```

Comment:
It would be cleaner to have a separate section in this function or even a separate mixin function to handle wordpress selectors.

Reply:
It was decided in today's we count check in to leave this section how it is because the wordpress selectors will adapt to the same style of the wecount site.